From 16fc166d3b1f048231fa4c504d4d0fb1725ddfa0 Mon Sep 17 00:00:00 2001
From: Philip Chimento <philip.chimento@gmail.com>
Date: Wed, 9 Nov 2016 22:36:52 -0800
Subject: build: Pass goa-1.0 dependency to VAPI build rule

This would otherwise fail due to not being able to find the symbols for
Goa.

https://bugzilla.gnome.org/show_bug.cgi?id=774179
---
 Makefile.am | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index e2ff442..fd3e754 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -605,6 +605,9 @@ gdata/libgdata.vapi: gdata/GData-@GDATA_API_VERSION_MAJOR@.@GDATA_API_VERSION_MI
 VAPIGEN_VAPIS = gdata/libgdata.vapi
 
 gdata_libgdata_vapi_DEPS = libxml-2.0 libsoup-2.4 json-glib-1.0
+if ENABLE_GOA
+gdata_libgdata_vapi_DEPS += goa-1.0
+endif
 gdata_libgdata_vapi_METADATADIRS = $(srcdir)/gdata
 gdata_libgdata_vapi_FILES = gdata/GData-@GDATA_API_VERSION_MAJOR@.@GDATA_API_VERSION_MINOR@.gir
 
-- 
cgit v0.12

